PWGMM/UE/Tasks/lambdak0sflattenicity.cxx

```diff
@@ -0,0 +1,826 @@
+    // calibration T0A
+    float calibT0A[24] = {0.86041, 1.10607, 1.17724, 0.756397, 1.14954, 1.0879, 0.829438, 1.09014, 1.16515, 0.730077, 1.06722, 0.906344, 0.824167, 1.14716, 1.20692, 0.755034, 1.11734, 1.00556, 0.790522, 1.09138, 1.16225, 0.692458, 1.12428, 1.01127};
+    // calibration factor MFT vs vtx
+    float biningVtxt[30] = {-14.5, -13.5, -12.5, -11.5, -10.5, -9.5, -8.5, -7.5, -6.5, -5.5, -4.5, -3.5, -2.5, -1.5, -0.5, 0.5, 1.5, 2.5, 3.5, 4.5, 5.5, 6.5, 7.5, 8.5, 9.5, 10.5, 11.5, 12.5, 13.5, 14.5};
+
+    // calibration factor FV0 vs vtx
+    float calibFV0vtx[30] = {0.907962, 0.934607, 0.938929, 0.950987, 0.950817, 0.966362, 0.968509, 0.972741, 0.982412, 0.984872, 0.994543, 0.996003, 0.99435, 1.00266, 0.998245, 1.00584, 1.01078, 1.01003, 1.00726, 1.00872, 1.01726, 1.02015, 1.0193, 1.01106, 1.02229, 1.02104, 1.03435, 1.00822, 1.01921, 1.01736};
+    // calibration FT0A vs vtx
+    float calibFT0Avtx[30] = {0.924334, 0.950988, 0.959604, 0.965607, 0.970016, 0.979057, 0.978384, 0.982005, 0.992825, 0.990048, 0.998588, 0.997338, 1.00102, 1.00385, 0.99492, 1.01083, 1.00703, 1.00494, 1.00063, 1.0013, 1.00777, 1.01238, 1.01179, 1.00577, 1.01028, 1.017, 1.02975, 1.0085, 1.00856, 1.01662};
+    // calibration FT0C vs vtx
+    float calibFT0Cvtx[30] = {1.02096, 1.01245, 1.02148, 1.03605, 1.03561, 1.03667, 1.04229, 1.0327, 1.03674, 1.02764, 1.01828, 1.02331, 1.01864, 1.015, 1.01197, 1.00615, 0.996845, 0.993051, 0.985635, 0.982883, 0.981914, 0.964635, 0.967812, 0.95475, 0.956687, 0.932816, 0.92773, 0.914892, 0.891724, 0.872382};
+
+    const int nDetVtx = 3;
+    TGraph* gVtx[nDetVtx];
+    const char* nameDet[nDetVtx] = {"AmpV0", "AmpT0A", "AmpT0C"};
+    for (int i_d = 0; i_d < nDetVtx; ++i_d) {
+      gVtx[i_d] = 0;
+      gVtx[i_d] = new TGraph();
+    }
+    for (int i_v = 0; i_v < 30; ++i_v) {
+      gVtx[0]->SetPoint(i_v, biningVtxt[i_v], calibFV0vtx[i_v]);
+    }
+    for (int i_v = 0; i_v < 30; ++i_v) {
+      gVtx[1]->SetPoint(i_v, biningVtxt[i_v], calibFT0Avtx[i_v]);
+    }
+    for (int i_v = 0; i_v < 30; ++i_v) {
+      gVtx[2]->SetPoint(i_v, biningVtxt[i_v], calibFT0Cvtx[i_v]);
+    }
+
+    for (int i_d = 0; i_d < nDetVtx; ++i_d) {
+      gVtx[i_d]->SetName(Form("g%s", nameDet[i_d]));
+    }
+
+    float sumAmpFV0 = 0;
+    float sumAmpFV01to4Ch = 0;
+    int innerFV0 = 32;
+    float maxEtaFV0 = 5.1;
+    float minEtaFV0 = 2.2;
+    float detaFV0 = (maxEtaFV0 - minEtaFV0) / 5.0;
+
+    const int nCells = 48; // 48 sectors in FV0
+    float ampchannel[48];
+    for (int iCell = 0; iCell < nCells; ++iCell) {
+      ampchannel[iCell] = 0.0;
+    }
+    float ampchannelBefore[48];
+    for (int iCell = 0; iCell < nCells; ++iCell) {
+      ampchannelBefore[iCell] = 0.0;
+    }
+
+    float RhoLattice[nCells];
```

Comment:
You can define this as an `std::array`, since the size is a static number, and outside the function. You can use its `fill` method to zero it. Same for other static arrays.

Reply:
The comments are now implemented. Thank you for the suggestion.